scripts/gtk-bundle-from-msys2.sh

```diff
@@ -104,6 +104,8 @@ handle_command_line_options() {
 }
 
```

Comment:
The patch clearly doesn't match the commit message

Reply:
I cannot see which patch and commit message you are referring to.